### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by Ola-Yeenca**
Theres a little css problems conflicting with the homepage from the login/signup page... its a bit complicated and I'll get help from the teacher later today

### CODE REVIEW

**Comment by Ola-Yeenca** [comment]
i will create a new pull request... just merge the master to my code.

**Comment by Ola-Yeenca** [comment]
done now

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -15,14 +15,9 @@
       <ul class="navbar-nav me-auto">
         <% if user_signed_in? %>
           <li class="nav-item dropdown">
-            <% if current_user.photo.present? %>
```
Comment (ThibaultPaulmann) [inline app/views/shared/_navbar.html.erb]: We shouldn't be deleting these file, they are essential for the avatar in the navbar that Mike did

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -15,14 +15,9 @@
...
-              <%= cl_image_tag current_user.photo.key, class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false%>
-            <% else %>
-              <%= image_tag "https://www.pngitem.com/pimgs/m/272-2720607_this-icon-for-gender-neutral-user-circle-hd.png", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
-            <% end %>
+            <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
             <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
 
-              <%= link_to "Edit profile", edit_user_registration_path, class: "dropdown-item" %>
```
Comment (ThibaultPaulmann) [inline app/views/shared/_navbar.html.erb:25]: you need to merge your thing with current master. If that doesn't work we'll have to manually rewrite this section

**Review by ThibaultPaulmann — CHANGES_REQUESTED** [review]
There is also something weird now with all the pages where we seem to have given it double margins and it is not finding the background video so the whole site crashes
Screenshot 2022-09-02 at 09 03 55

**Comment by Ola-Yeenca** [comment]
I left a message on slack... with the video

**Pull request closed without merging**